fluence-app-service/src/config.rs

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use std::path::PathBuf;
+
+/// Describes behaviour of the Fluence AppService.
+#[derive(Default)]
+pub struct AppServiceConfig {
+    /// Used for preparing filesystem on the service initialization stage.
+    pub service_base_dir: PathBuf,
+
+    pub faas_config: fluence_faas::FaaSConfig,
+}
```

Comment:
I'd move `fluence_faas::FaaSConfig` to imports.

```suggestion
use std::path::PathBuf;
use fluence_faas::FaaSConfig;

/// Describes behaviour of the Fluence AppService.
#[derive(Default)]
pub struct AppServiceConfig {
    /// Used for preparing filesystem on the service initialization stage.
    pub service_base_dir: PathBuf,
    pub faas_config: FaaSConfig,
}
```